staging/src/k8s.io/api/rbac/v1/types.go

```diff
@@ -106,6 +106,7 @@ type RoleRef struct {
 
 // +genclient
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +k8s:prerelease-lifecycle-gen:introduced=1.7
```

Comment:
Role, RoleBinding, ClusterRole and ClusterRole binding should be 1.8?

I checked out the release-1.7 and release-1.8 branches and grepped the types.go files and see these introduced in 1.8.

Reply:
Changed